packages/ui-demo/src/Card.stories.tsx

```diff
@@ -27,13 +27,19 @@ export const CardStories = {
                 rounding="circle"
                 width={50}
               >
-                <h2>JG</h2>
+                <h2>
+                  <Text>JG</Text>
+                </h2>
               </Box>
               <Box direction="column" paddingX={2}>
                 <div>
-                  <b>Josh Gachnang</b>
```

Comment:
Ah ha, good catch. I'm surprised this ever ran on iOS/Android with HTML elements though